app/src/main/java/org/oppia/android/app/help/thirdparty/LicenseTextViewerActivity.kt

```diff
@@ -0,0 +1,41 @@
+package org.oppia.android.app.help.thirdparty
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+/** The activity that will show the license text of a copyright license. */
+class LicenseTextViewerActivity : InjectableAppCompatActivity() {
+
+  @Inject
+  lateinit var licenseTextViewerActivityPresenter: LicenseTextViewerActivityPresenter
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    activityComponent.inject(this)
+    val dependencyIndex = intent.getIntExtra(LICENSE_TEXT_VIEWER_ACTIVITY_DEP_INDEX, 1)
+    val licenseIndex = intent.getIntExtra(LICENSE_TEXT_VIEWER_ACTIVITY_LICENSE_INDEX, 1)
+    licenseTextViewerActivityPresenter.handleOnCreate(dependencyIndex, licenseIndex)
+  }
+
+  companion object {
+    /** Returns [Intent] for [LicenseTextViewerActivity]. */
+    private const val LICENSE_TEXT_VIEWER_ACTIVITY_DEP_INDEX =
+      "LicenseTextViewerActivity.DependencyIndex"
```

Comment:
Nit ` "LicenseTextViewerActivity.dependency_index"`

Same for other places

Reply:
Done